webapp/pages/StatisticalFactsheets/ForestOwnership/ForestOwnership.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Areas } from '@core/country'
-import Chart from '@webapp/pages/StatisticalFactsheets/components/chart/Chart'
-import * as ChartUtils from '@webapp/pages/StatisticalFactsheets/utils/chartUtils'
+import Chart from '@webapp/components/Chart/Chart'
+import * as ChartUtils from '@webapp/components/Chart/chartUtils'
```

Comment:
ChartUtils can become ChartData. and can be exported from chart/index. 
ChartUtils can be split by responsibility (Single responsibility is a must)